cmd/minishift/cmd/start.go

```diff
@@ -35,15 +39,14 @@ import (
 	minishiftConfig "github.com/minishift/minishift/pkg/minishift/config"
 	"github.com/minishift/minishift/pkg/minishift/hostfolder"
 	"github.com/minishift/minishift/pkg/minishift/provisioner"
+	"github.com/minishift/minishift/pkg/minishift/registration"
 	"github.com/minishift/minishift/pkg/util"
+	pkgUtil "github.com/minishift/minishift/pkg/util"
```

Comment:
I see the ambiguity to minishift/util, especially if you would try to use both in the same go file. Happy to use something else as well. pkgUtil seems wrong, since it seems to be based on the _pkg_ directory in which it is located, but that is not really part of the package names. 

_generalUtils_? _inputUtils_ (since this is the context in which you are using this in this case)?